exercises/sieve/test/test_sieve.c

```diff
@@ -40,14 +37,14 @@ void test_find_first_prime(void)
 
    result_prime_count = sieve(limit, result_array);
    TEST_ASSERT_EQUAL(expected_prime_count, result_prime_count);
-   TEST_ASSERT_EQUAL(0,
-                     memcmp(expected_prime_array, result_array,
-                            sizeof(expected_prime_array)));
+   TEST_ASSERT_EQUAL_UINT_ARRAY(expected_prime_array, result_array,
+                                ARRAY_LENGTH(expected_prime_array));
+
```

Comment:
Can you remove this extra whitespace here and in the other tests?

Reply:
CI fails when I delete it, it's put there by indent.sh.  I can try tackling an indent update if we want, but it might take a little while.

Reply:
Let's not fight the formatter. It certainly makes odd decisions sometimes, though.